### ISSUE TITLE
Rename "Guidance scale" to "CFG scale" with <abbr> tag

### ISSUE THREAD

**Issue by ssokolow** [easydiffusion/easydiffusion#1684]
**Is your feature request related to a problem? Please describe.**

When I first started using Easy Diffusion, I wanted to start my experiments by replicating results on Civitai and then working from there. It took me an inordinately long time to figure out what Easy Diffusion calls certain settings and where it keeps them, with "CFG scale" being one.

**Describe the solution you'd like**

Instead of the vague and different "Guidance scale" (guidance of what? for what?), use `<abbr title="Classifier-Free Guidance">CFG</abbr> scale` and make sure `<abbr>` still has the dotted underline it gets by default. 

This will harmonize it with the name used elsewhere, making learning materials written with another distribution in mind more accessible to Easy Diffusion users while still providing access to what "CFG" means for those who want to know. As an added benefit, it makes the *full* name of the parameter available without needing to search.

**Describe alternatives you've considered**
I considered leaving "Guidance Scale" as-is and putting "CFG Scale" in a tooltip, but that doesn't make as much logical sense and putting an abbreviation in a tooltip doesn't have the first-class support that `<abbr>` offers for going the other way.

I also considered adding a question mark icon, but I don't think that would have helped because I wasn't looking for what "Guidance Scale" meant, I was scanning through Image Settings looking for the string "CFG scale" or at least the acronym "CFG".

**Comment by ssokolow** [comment]
True, but I'm suggesting the rename to resolve the "Where the heck is CFG scale!?[/frustrated]" problem with following basically every piece of learning material I've encountered so far.

**Comment by fdwr** [comment]
> guidance of what? for what?
The "guidance scale" (or if you want the overly long term "classifier free guidance scale") guides the image in the direction away from the negative prompt and guides toward the positive prompt.

> making learning materials ... more accessible
If your goal is to make learning materials more accessible to beginners, then avoiding cryptic acronyms like CFG would be a good start. If anything, wouldn't it be a nicer outcome to petition Civitai and A1111 to change "CFG" to "Guidance"?

**Comment by ssokolow** [comment]
> The "guidance scale" (or if you want the overly long term "classifier free guidance scale") guides the image in the direction away from the negative prompt and guides toward the positive prompt.

"guidance of what? for what?" is what users like me said when we looked at the UI for the first time. Telling me here doesn't solve the problem because there's no easy way for future questioners to get from there to here.

> If your goal is to make learning materials more accessible to beginners, then avoiding cryptic acronyms like CFG would be a good start. If anything, wouldn't it be a nicer outcome to petition Civitai and A1111 to change cryptic "CFG" to "Guidance"?

Guidance and CFG are both cryptonyms without additional online help, like a tooltip or (i) button, and I don't think there's a non-cryptonym good enough and short enough to be displayed as the field's label, but CFG is more compatible with people wanting to follow instructions written with automatic1111 in mind.

**Comment by fdwr** [comment]
> without additional online help, like a tooltip ...
> Guidance and CFG are both cryptonyms

Yeah, extra information is prudent for both the even more confusing A1111 term (what the flip is "CFG", said the first time user??) and Easy Diffusion. Want to open an A1111 [issue here](https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues)? Tooltips even have first-class support too in HTML (no extra JS needed) by just putting a `title` element on a div ([e.g.](https://stackoverflow.com/a/7117107/937938)).

Note in the actual math, the balance between the positive and negative prompt guides the prediction further along each iteration, either more strictly (more to the positive weight) or less strictly (to the negative weight):

```
negativeWeight = 1 - positiveWeight; // e.g. 7.5 positive weight yields -6.5 negative weight
...
predictedNoise[i] = predictedNoise[i] * positiveWeight
                  + predictedNoise[i + secondBatch] * negativeWeight;
```

Personally I always found the term "classifier free guidance" to be a mouthful, which is a nice lengthy word suitable for an ML research paper, but it's not helpful to users. Really it is simply the *positive prompt weight*.

**Comment by ssokolow** [comment]
I agree that it's *all* full of opaque research paper naming, but my point is that it's too late to change the forest of guides and other sites that call it "CFG scale" so, if both are problematic, might as well apply some HCI 101 and treat it like learning an icon.

(As my introductory HCI textbook said, affordances like "this icon is a trash can" only help while learning a UI. Once the user has learned the UI, consistency with other interfaces, such as "the save icon should be a floppy disk, despite nobody using them anymore, because it's become 'the save glyph'" is paramount.)

Tooltip or (i) for teaching what "CFG scale" means; "CFG scale" for consistency with all the guides that have been written.

**Comment by fdwr** [comment]
> Aside from its apparent lack of JSON import/export for generation parameters making me disinterested

:(

> I also almost never run automatic1111 on my current machine

And Easy Diffusion is much nicer for exploring variations and seeing your current queue and past generations.

**Comment by ssokolow** [comment]
> And Easy Diffusion is much nicer for exploring variations and seeing your current queue and past generations.

Exactly. In my opinion, Easy Diffusion has the superior UI design without question. The only reasons I sometimes use automatic1111 are:

1. Support for LoCon
2. The plugin that lets you interactively create/edit OpenPose ControlNets

**Comment by cmdr2** [comment]
Yeah, merging https://github.com/easydiffusion/easydiffusion/issues/1937 into this.

Basically, ED needs to communicate that:
- Guidance scale is another name for "CFG Scale"
- Prompt strength is another name for "Denoising strength"
